osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -244,15 +230,9 @@ private void load(OsuConfigManager config, AudioManager audio, CancellationToken
 
             RightClickScrollingEnabled.BindValueChanged(enabled => Scroll.RightMouseScrollbar = enabled.NewValue, true);
 
-            if (detachedBeatmapStore != null && detachedBeatmapSets == null)
-            {
-                // This is performing an unnecessary second lookup on realm (in addition to the subscription), but for performance reasons
-                // we require it to be separate: the subscription's initial callback (with `ChangeSet` of `null`) will run on the update
-                // thread. If we attempt to detach beatmaps in this callback the game will fall over (it takes time).
```

Comment:
This comment should probably remain in source? I don't see any reason why it would no longer be valid.

Reply:
Discussed IRL, comment should have been deleted in 466ed5de785e2f2f70a5077bdb8f1d527aad788d